samples/net/ocpp/prj.conf

```diff
@@ -0,0 +1,46 @@
+# HTTP & Websocket
+CONFIG_HTTP_CLIENT=y
+CONFIG_WEBSOCKET_CLIENT=y
+
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_TCP=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POLL_MAX=4
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_ETH_DRIVER=y
+CONFIG_LOG=y
+CONFIG_NET_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=3
+
+# OCPP
+CONFIG_OCPP=y
+CONFIG_OCPP_SERVER="ssh.linumiz.com"
+CONFIG_OCPP_PORT=8180
+
+CONFIG_SNTP_SERVER="162.159.200.1"
```

Comment:
Also here, everything specific to your own installation should not be mentioned here. Just leave the server and sntp server empty or commented out. You can always set these separately when compiling using west.

Reply:
These values are public reachable, anyone can directly run sample app without any changes. Please suggest if you still consider to remove it

Reply:
No need to remove the options, just set empty values something like this.
```
# Please set the server addresses when compiling the sample
CONFIG_SNTP_SERVER=""
CONFIG_OCPP_SERVER=""
CONFIG_OCPP_PORT=8180
```